BuildResidentialHPXML/measure.rb

```diff
@@ -3838,13 +3868,18 @@ def self.set_heating_systems(hpxml, runner, args)
       heating_efficiency_percent = args[:heating_system_heating_efficiency]
     end
 
+    if args[:heating_system_airflow_defect_ratio].is_initialized
+      airflow_defect_ratio = args[:heating_system_airflow_defect_ratio].get
```

Comment:
Should only be setting this for the system types that it applies to, as described above. Same for all new inputs below.